dj_rest_auth/registration/views.py

```diff
@@ -112,23 +112,19 @@ def post(self, request, *args, **kwargs):
 class ResendEmailVerificationView(CreateAPIView):
     permission_classes = (AllowAny,)
     serializer_class = ResendEmailVerificationSerializer
+    queryset = EmailAddress.objects.all()
 
     def create(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
 
-        email = EmailAddress.objects.get(**serializer.validated_data)
-        if not email:
-            raise ValidationError("Account does not exist")
+        email = EmailAddress.objects.filter(**serializer.validated_data).first()
```

Comment:
This line could use `self.queryset.filter(...)`, so that if a subclass changes the queryset attribute, that wouldn’t be ignored.